index.js

```diff
@@ -294,12 +293,16 @@ gulp.task('index_html_build', function() {
         .pipe(gulp.dest('src'));
 });
 
+var rewriteModule = require('http-rewrite-middleware');
+var rewriteMiddleware = rewriteModule.getMiddleware(config.rewriteMiddleware);
```

Comment:
no need for intermediate var, we can just throw this right into the webserver opts right?